pandas/core/window/rolling.py

```diff
@@ -591,6 +599,79 @@ def _apply(
         result.index = result_index
         return result
 
+    def _apply_pairwise(self, target, other, pairwise, func):
```

Comment:
wow. can you type the input / output args and add a doc-string.

Reply:
Done (a testament to how complex groupby apply is)